src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -21,12 +22,18 @@ interface ProductsTableProps {
   filter: string;
   setFilter(filter: string): void;
 }
+const useQuery = () => {
+  return new URLSearchParams(useLocation().search);
+};
```

Comment:
Suggestion: the SubscriptionInventoryPage component handles all the filtering so we shouldn't need this